ddl/index.go

```diff
@@ -1922,7 +1928,39 @@ func (w *worker) executeDistGlobalTask(reorgInfo *reorgInfo) error {
 			}
 		}
 	})
-	return g.Wait()
+	err = g.Wait()
+	return err
+}
+
+func (w *worker) updateJobRowCount(taskKey string, jobID int64) {
+	taskMgr, err := storage.GetTaskManager()
+	if err != nil {
+		logutil.BgLogger().Error("cannot get task manager", zap.String("category", "ddl"), zap.String("task_key", taskKey), zap.Error(err))
+		return
+	}
+	gTask, err := taskMgr.GetGlobalTaskByKey(taskKey)
+	if err != nil || gTask == nil {
+		logutil.BgLogger().Error("cannot get global task", zap.String("category", "ddl"), zap.String("task_key", taskKey), zap.Error(err))
+		return
+	}
+	metas, err := taskMgr.GetSucceedSubtaskMeta(gTask.ID)
```

Comment:
Can we add a field in `tidb_background_subtask` to record the rowCount the subtask processed?
Then the marshal and unmarshal and count rowCount process will be eliminated.
We can discussion it later.

Reply:
https://github.com/pingcap/tidb/pull/46562